corehq/apps/export/views.py

```diff
@@ -80,11 +77,6 @@ def export_home_url(self):
     @memoized
     def report_class(self):
         try:
-            if toggle_enabled(self.request, toggles.REVAMPED_EXPORTS):
```

Comment:
no, this is just for the redirect and this is just for the REVAMPED exports flag for the first round of revamping (I actually am using that toggle for the new revamp and handle this differently, so it doesn't need to be removed elsewhere)